docs/DeveloperGuide.md

```diff
@@ -304,6 +304,72 @@ results based on the specified criteria.
 
 <puml src="diagrams/FindSequenceDiagram.puml" alt="FindSequenceDiagram" />
 
+## Export feature
+
+### Implementation
+
```

Comment:
Shall we add a detail of export command inherit from fileAccessCommand, well not necessary lah